msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -25,8 +25,8 @@ The rest of this guide walks you through these steps and shows how to preview yo
 To complete this tutorial, you need:
 
 * A Microsoft 365 Developer Program sandbox tenant.
-* Enrollment in *Office 365 Targeted Releases* for your sandbox tenant.
-* A test environment with Office apps installed from the Microsoft 365 Apps *Beta Channel*.
+* Enrollment in *Microsoft 365 Targeted Releases* for your sandbox tenant.
```

Comment:
Looks like this is still called "Office 365 Targeted Releases" in the admin center